<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qt4ProjectManager::Internal::QtVersionManager</class>
 <widget class="QWidget" name="Qt4ProjectManager::Internal::QtVersionManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>477</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Qt versions</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QTreeWidget" name="qtdirList">
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Path</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Debugging Helper</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="2">
       <layout class="QVBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="addButton">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="delButton">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="versionNameLabel">
        <property name="text">
         <string>Version Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="nameEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="pathLabel">
        <property name="text">
         <string>Path:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="Core::Utils::PathChooser" name="qtPath" native="true"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="mingwLabel">
        <property name="text">
         <string>MinGw Directory:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="Core::Utils::PathChooser" name="mingwPath" native="true"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="msvcLabel">
        <property name="text">
         <string>MSVC Version:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="msvcComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="msvcNotFoundLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Unable to detect MSVC version.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Debugging Helper:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="debuggingHelperStateLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="showLogButton">
          <property name="text">
           <string>Show &amp;Log</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rebuildButton">
          <property name="text">
           <string>&amp;Rebuild</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="errorLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="defaultLabel">
       <property name="text">
        <string>Default Qt Version:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="defaultCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Core::Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>qtdirList</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>delButton</tabstop>
  <tabstop>nameEdit</tabstop>
  <tabstop>qtPath</tabstop>
  <tabstop>mingwPath</tabstop>
  <tabstop>msvcComboBox</tabstop>
  <tabstop>showLogButton</tabstop>
  <tabstop>rebuildButton</tabstop>
  <tabstop>defaultCombo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
